AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -805,8 +984,19 @@ def run_fbx_test(self, workspace, ap_setup_fixture, asset_processor,
                 for expected_product in expected_job.products:
                     expected_product_list.append(expected_product.product_name)
 
+        cache_folder = get_cache_folder(asset_processor)
         missing_assets, _ = utils.compare_assets_with_cache(expected_product_list,
-                                                            asset_processor.project_test_cache_folder())
+                                                            cache_folder)
+
+        # If the test is going to fail, print information to help track down the cause of failure.
+        if missing_assets:
+            logger.info(f"The following assets were missing from cache:")
+            for asset in missing_assets:
```

Comment:
nit: Did you consider using pprint.fomat or similar to avoid loop?

Reply:
Done, added